src/effects.cpp

```diff
@@ -259,6 +259,8 @@ unique_ptr<LEDStripEffect> GetSpectrumAnalyzer(CRGB color)
 }
 #endif
 
+CRGB mult = CRGB::Black; float prob = 0.9; // These constants are referenced by several project's effects

```

Comment:
I'm wondering if setting these variables to fixed values in the cpp file is the best way to go. Maybe it would make sense to:
1. set their values using header defines,or
2. replace the references to the variables with such defines throughout effects.cpp, or even
3. replacing the references with the constants entirely?

Reply:
Yup. I was thinking this wasn't a great solution. Though, the values that are used for these variables are probably highly dependent on preference, and the hardware used.

Reply:
As they are used in multiple projects' effects, it may make most sense to add a couple of defines to globals.h and use those in effects.cpp; that header already contains some "semi-configuration", anyway.

Reply:
I agree
As I said before, the way I implemented those variables was for testing purposes only.
I, too, would add the defines to the globals header, and also add some default values, in case a project/config doesn't want to override it

Reply:
Sounds good. I'll make the changes and force push when I have a chance.